core/unit_test/TestTeamReductionScan.hpp

```diff
@@ -52,18 +56,70 @@ TEST(TEST_CATEGORY, team_double_reduce) {
                               Kokkos::Experimental::OpenMPTarget>::value)
 #endif
   {
-    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Static> >(
-        0);
-    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Dynamic> >(
-        0);
-    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Static> >(
-        3);
-    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Dynamic> >(
-        3);
-    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Static> >(
-        100000);
-    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Dynamic> >(
-        100000);
+    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Static> >{}
+        .run_test(0);
+    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Dynamic> >{}
+        .run_test(0);
+    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Static> >{}
+        .run_test(3);
+    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Dynamic> >{}
+        .run_test(3);
+    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Static> >{}
+        .run_test(100000);
+    TestReduceTeam<double, TEST_EXECSPACE, Kokkos::Schedule<Kokkos::Dynamic> >{}
+        .run_test(100000);
+  }
+}
+
+TEST(TEST_CATEGORY, team_long_array_reduce) {
+// FIXME_MSVC FIXME_32BIT Test is known to fail
+#if !defined(KOKKOS_COMPILER_MSVC) && !defined(KOKKOS_IMPL_32BIT)
+  GTEST_SKIP() << "Test know to fail for MSVC or 32-bit builds";
+#endif
+
+#ifdef KOKKOS_ENABLE_OPENMPTARGET  // FIXME_OPENMPTARGET: Not implemented
+  if constexpr (!std::is_same<TEST_EXECSPACE,
+                              Kokkos::Experimental::OpenMPTarget>::value)
+#endif
```

Comment:
Uggg
The gtest output will suggest the test run.  Not blocking considering the issue was already there